### ISSUE TITLE
Load Error while importing Images

### ISSUE THREAD

**Issue by kvmanohar22** [JuliaImages/Images.jl#604]
System setup : 
```
$ uname -a
Linux kv 3.19.0-78-generic #86~14.04.1-Ubuntu SMP Tue Dec 6 17:58:35 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux


```
I use julia `0.6.0-pre.alpha`
Error : 
```julia
julia> using Images

WARNING: deprecated syntax "typealias AbstractGray{T} Color{T,1}" at /home/kv/.julia/v0.6/Images/src/Images.jl:38.
Use "AbstractGray{T} = Color{T,1}" instead.

WARNING: deprecated syntax "typealias TransparentRGB{C<:AbstractRGB,T} TransparentColor{C,T,4}" at /home/kv/.julia/v0.6/Images/src/Images.jl:39.
Use "TransparentRGB{C<:AbstractRGB,T} = TransparentColor{C,T,4}" instead.

WARNING: deprecated syntax "typealias TransparentGray{C<:AbstractGray,T} TransparentColor{C,T,2}" at /home/kv/.julia/v0.6/Images/src/Images.jl:40.
Use "TransparentGray{C<:AbstractGray,T} = TransparentColor{C,T,2}" instead.

WARNING: deprecated syntax "typealias NumberLike Union{Number,AbstractGray}" at /home/kv/.julia/v0.6/Images/src/Images.jl:41.
Use "const NumberLike = Union{Number,AbstractGray}" instead.

WARNING: deprecated syntax "typealias RealLike Union{Real,AbstractGray}" at /home/kv/.julia/v0.6/Images/src/Images.jl:42.
Use "const RealLike = Union{Real,AbstractGray}" instead.
INFO: Precompiling module Images.

WARNING: deprecated syntax "typealias AbstractGray{T} Color{T,1}" at /home/kv/.julia/v0.6/Images/src/Images.jl:38.
Use "AbstractGray{T} = Color{T,1}" instead.

WARNING: deprecated syntax "typealias TransparentRGB{C<:AbstractRGB,T} TransparentColor{C,T,4}" at /home/kv/.julia/v0.6/Images/src/Images.jl:39.
Use "TransparentRGB{C<:AbstractRGB,T} = TransparentColor{C,T,4}" instead.

WARNING: deprecated syntax "typealias TransparentGray{C<:AbstractGray,T} TransparentColor{C,T,2}" at /home/kv/.julia/v0.6/Images/src/Images.jl:40.
Use "TransparentGray{C<:AbstractGray,T} = TransparentColor{C,T,2}" instead.

WARNING: deprecated syntax "typealias NumberLike Union{Number,AbstractGray}" at /home/kv/.julia/v0.6/Images/src/Images.jl:41.
Use "const NumberLike = Union{Number,AbstractGray}" instead.

WARNING: deprecated syntax "typealias RealLike Union{Real,AbstractGray}" at /home/kv/.julia/v0.6/Images/src/Images.jl:42.
Use "const RealLike = Union{Real,AbstractGray}" instead.
WARNING: could not import Base.slice into Images
WARNING: could not import Base.sub into Images

WARNING: deprecated syntax "abstract MapInfo{T}" at /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:51.
Use "abstract type MapInfo{T} end" instead.

WARNING: deprecated syntax "abstract AbstractClamp{T}<:MapInfo{T}" at /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:139.
Use "abstract type AbstractClamp{T}<:MapInfo{T} end" instead.

WARNING: deprecated syntax "typealias GrayArray{T<:Union{Fractional,Bool}} Union{AbstractArray{T},AbstractArray{Gray{T}}}" at /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:588.
Use "GrayArray{T<:Union{Fractional,Bool}} = Union{AbstractArray{T},AbstractArray{Gray{T}}}" instead.

WARNING: deprecated syntax "inner constructor MapNone(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:62".
Use "MapNone{T}(...) where T" instead.

WARNING: deprecated syntax "inner constructor BitShift(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:114".
Use "BitShift{T,N}(...) where {T,N}" instead.

WARNING: deprecated syntax "inner constructor ClampMin(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:149".
Use "ClampMin{T,From}(...) where {T,From}" instead.

WARNING: deprecated syntax "inner constructor ClampMax(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:165".
Use "ClampMax{T,From}(...) where {T,From}" instead.

WARNING: deprecated syntax "inner constructor ClampMinMax(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:176".
Use "ClampMinMax{T,From}(...) where {T,From}" instead.

WARNING: deprecated syntax "inner constructor Clamp(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:199".
Use "Clamp{T}(...) where T" instead.

WARNING: deprecated syntax "inner constructor ScaleMinMax(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:265".
Use "ScaleMinMax{To,From,S}(...) where {To,From,S}" instead.

WARNING: deprecated syntax "inner constructor ScaleSigned(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:334".
Use "ScaleSigned{T,S}(...) where {T,S}" instead.

WARNING: deprecated syntax "inner constructor ScaleAutoMinMax(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:365".
Use "ScaleAutoMinMax{T}(...) where T" instead.

WARNING: deprecated syntax "inner constructor ScaleMinMaxNaN(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:385".
Use "ScaleMinMaxNaN{To,From,S}(...) where {To,From,S}" instead.

WARNING: deprecated syntax "inner constructor Clamp01NaN(...) around /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:397".
Use "Clamp01NaN{T}(...) where T" instead.
ERROR: LoadError: LoadError: type UnionAll has no field abstract
Stacktrace:
 [1] macro expansion at /home/kv/.julia/v0.6/Images/src/map-deprecated.jl:431 [inlined]
 [2] anonymous at ./<missing>:?
 [3] include_from_node1(::String) at ./loading.jl:539
 [4] include(::String) at ./sysimg.jl:14
 [5] include_from_node1(::String) at ./loading.jl:539
 [6] include(::String) at ./sysimg.jl:14
 [7] anonymous at ./<missing>:2
while loading /home/kv/.julia/v0.6/Images/src/map-deprecated.jl, in expression starting on line 429
while loading /home/kv/.julia/v0.6/Images/src/Images.jl, in expression starting on line 60
ERROR: Failed to precompile Images to /home/kv/.julia/lib/v0.6/Images.ji.
Stacktrace:
 [1] compilecache(::String) at ./loading.jl:673
 [2] require(::Symbol) at ./loading.jl:460

julia> 

```

**Comment by timholy** [comment]
Images won't fully work with 0.6 until StaticArrays.jl is 0.6-ready. That said, try the `teh/0.6` branch.

**Comment by kvmanohar22** [comment]
Thanks ! that works.

**Comment by timholy** [comment]
Closed by #614
